functions.gallery.php

```diff
@@ -33,10 +42,15 @@ function wp_enqueue_media() {
 		wp_enqueue_script( 'jetpack-gallery-settings' );
 	}
 
-	/**
-	 * Outputs a view template which can be used with wp.media.template
-	 */
 	function print_media_templates() {
+		/**
+		 * Filter the default gallery type.
+		 *
+		 * @since 2.5.1
+		 *
+		 * @param array  $instance A string of gallery types.
```

Comment:
Since this is a string, you'd need to change the type of that parameter from `array` to `string`.

"A string of gallery types." should also probably be "A string of the default gallery type."